### PR TITLE
Add the option not to open the browser automatically

### PR DESCRIPTION
**Pull request by mvochoa**
I have added this option because my computer has a graphical interface, but I only have access via SSH

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ Run `go install github.com/JorgeReus/aws-sso-creds/cmd/aws-sso-creds@latest`
 Usage: aws-sso-creds [-fhpst] [-r value] [-u value] [parameters ...]
  -f, --forceAuth    Force Authentication with AWS SSO
  -h, --help         Help
+     --noInteractive
```
Comment (JorgeReus) [inline README.md]: Replace *no* with *non* in noInteractive (i.e. nonInteractive) or even replace it with noBrowser

File: cmd/aws-sso-creds/main.go [hunk 2]
```diff
@@ -19,6 +19,7 @@ var (
 	ssoURL                  = getopt.StringLong("SSOUrl", 'u', "", "The SSO URL(https://<my-domain>.awsapps.com/start) the AWS_SSO_START_URL env var has precedence over this flag")
 	ssoRegion               = getopt.StringLong("SSORegion", 'r', "", "The AWS SSO region, the AWS_SSO_REGION env var has precedence over this flag")
 	forceLogin              = getopt.BoolLong("forceAuth", 'f', "Force Authentication with AWS SSO")
+	noInteractive           = getopt.BoolLong("noInteractive", 0, "Do not open in the browser automatically")
```
Comment (JorgeReus) [inline cmd/aws-sso-creds/main.go]: Replace no with non in noInteractive (i.e. nonInteractive) or even replace it with noBrowser (Duplicate)

**Review by JorgeReus — CHANGES_REQUESTED** [review]
Hey there!
Thanks for contributing, left some comments, feel free to comment other changes!

File: cmd/aws-sso-creds/main.go (hunk 2 above)
Comment (mvochoa) [reply cmd/aws-sso-creds/main.go]: Done I made the change

**Comment by JorgeReus** [comment]
LGTM!

**Comment by JorgeReus** [comment]
Released new version, thanks!
https://github.com/JorgeReus/aws-sso-creds/releases/tag/1.0.3

**Pull request merged**